index.html

```diff
@@ -80,6 +80,10 @@
         <button data-action="log-data">Log segments/points</button>
       </div>
 
+      <ul id="segments">
```

Comment:
The layout of the demo page needs some work, to align the segments with the other page content. I'm thinking that instead of showing each segment as it's added we should change the `log segments/points` button to show the segments (and points) in the page, rather than in the console.